README.md

```diff
@@ -163,12 +167,12 @@ curl -X 'PUT' \
 
 Also process flows must be added to the chain with `npm run flows`. Process flows validate transactions that affect the chain.
 
-1. `MemberA` wants to create a `capacity`, which includes a parameters file to the parameters of the available capacity they have. The parameters file will be used by `Optimiser` when matching `capacity` with a `order`. First `MemberA` must upload this parameters file to their local database with `POST /attachment`.
-2. They use the returned `id` for `parametersAttachmentId` in the request body to `POST /capacity`. At this point, the `capacity` only exists in the `MemberA` database.
-3. When `MemberA` is ready for the `capacity` to exist on chain they `POST capacity/{capacityId}/creation`. `MemberB` and `Optimiser` can now see the `capacity` if their node is running and connected.
-4. `MemberB` creates an `order` in a similar manner to creating a `capacity`. It includes a parameters file to describe the parameters of their order.
-5. When `MemberB` is ready for the `order` to exist on chain they `POST order/{id}/creation`.
-6. `Optimiser` can now create a `match2` that matches a single `capacity` with a single `order`. They supply these as an `id` for `demandA` and `demandB`. It doesn't matter which is `demandA` and which is `demandB`.
+1. `MemberA` wants to create a `demandA`, which includes a parameters file to the parameters of the available demandB they have. The parameters file will be used by `Optimiser` when matching `demandA` with a `demandA`. First `MemberA` must upload this parameters file to their local database with `POST /attachment`.
```

Comment:
demandA/demandB mixing. Guessing should be `available demandB` and `matching demandA with a demandB`